### ISSUE TITLE
[MU4 Task] Instrument selective playback should also work with multistaff instruments (Piano, Divisi strings etc.) 

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#12230]
**Task description**

https://user-images.githubusercontent.com/90187801/176744468-3f08a606-f22e-46ea-9467-65816a98c629.mp4

**Comment by s1m0n-github** [comment]
I've stumbled across an issue related to this when testing, but as demonstrated here the full score playback works when selecting the note/break instead of the whole measure. Is this by design or is it an issue with the selective playback? I find it quite disturbing to always have to aim for a small note/break when I want to play my score back from somewhere

**Comment by DmitryArefiev** [comment]
> but as demonstrated here the full score playback works when selecting the note/break instead of the whole measure. Is this by design or is it an issue with the selective playback?

@s1m0n-github Yes, it's by design in MuseScore 4

**Comment by s1m0n-github** [comment]
Thanks, I'll have to get used to that.

**Comment by DmitryArefiev** [comment]
Yes that's new to MuseScore4, but it also gives a good ability to listen instrument lines separately
